ebpf_extensions/neteventebpfext/netevent_ebpf_ext_event.c

```diff
@@ -63,12 +63,26 @@ _netevent_ebpf_extension_detach_provider(_In_ HANDLE nmr_binding_handle);
 // Structures for attaching to NetEvent (as an NMR client)
 //
 
+typedef struct netevent_ext_header
```

Comment:
Why are we duplicating this type? Seems like this should be a common one.

Reply:
This header is specifically for the NMI interface between netevent ext and internal netevent provider. Using the common (ebpf) header leads to crashes when integrating with the netevent provider.